src/ImpactEvaluator.sol

```diff
@@ -6,17 +6,15 @@ pragma solidity ^0.8.19;
 contract ImpactEvaluator is AccessControl {
     struct Round {
         uint end;
-        string[] measurementsCids;
         address payable[] addresses;
         uint64[] scores;
```

Comment:
we need to temporarily keep these now, for the case where the evaluator submits scores over multiple calls, and we need to merge them afterwards